Source: maxice8
Upstream: no
Reason: fixes compilation by avoiding using __ptrace_request as an enum.

--- a/plugins/DebuggerCore/unix/linux/arch/x86-generic/PlatformThread.cpp
+++ b/plugins/DebuggerCore/unix/linux/arch/x86-generic/PlatformThread.cpp
@@ -33,6 +33,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <sys/ptrace.h>
 #include <sys/user.h>
 
+#ifndef __GLIBC__
+#define __ptrace_request int
+#endif
+
 // doesn't always seem to be defined in the headers
 #ifndef PTRACE_GET_THREAD_AREA
